geomesa-nifi-processors/src/main/scala/org/geomesa/nifi/geo/validators/ConverterValidator.scala

```diff
@@ -0,0 +1,21 @@
+package org.geomesa.nifi.geo.validators
+
+import org.apache.nifi.components.{ValidationContext, ValidationResult, Validator}
+import org.locationtech.geomesa.convert.{ConfArgs, ConverterConfigResolver}
+
+/**
+  * Simple validator of a convert spec. Tries to parse it to make sure that it can.
+  */
+class ConverterValidator extends Validator{
+  override def validate(subject: String, input: String, context: ValidationContext): ValidationResult = {
+    ConverterConfigResolver.getArg(ConfArgs(input)) match {
+      case Left(_) => new ValidationResult.Builder().subject(subject).explanation(String.format("'%s' is not a valid converter", subject)).input(input).build;

```

Comment:
[minor] can use scala string interpolation instead of string.format: `s"'$subject' is not a valid converter"`